x-pack/plugins/security_solution/public/timelines/components/side_panel/user_details/index.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable react/display-name */
```

Comment:
is there a reason?

Reply:
No, I missed that during copy and paste. I will name the component.